tasks/build.py

```diff
@@ -359,7 +396,20 @@ def prepare_jobs(pr, event_dir, build_env_cfg, arch_map, repocfg):
             if repo_id != "EESSI-pilot" and repo_id not in repocfg:
                 log(f"{fn}(): skipping repo {repo_id}, it is not defined in repo config {repocfg[REPOS_CFG_DIR]}")
                 continue
-            job_dir = os.path.join(event_dir, arch_dir, repo_id)
+
+            # if filter exists, check filter against context = (arch, repo, app_name)
+            #   true --> log & go on
+            #   false --> log & continue
```

Comment:
`log & continue` to next iteration of `for` loop

Reply:
done